packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -280,10 +280,9 @@ function GlobalStylesEditorCanvasContainerLink() {
 				 * Exclude revisions panel from this behavior,
 				 * as it should close when the editorCanvasContainerView doesn't correspond.
 				 */
-				if ( path !== '/' && ! isRevisionsOpen ) {
-					return;
+				if ( path !== '/' && isRevisionsOpen ) {
+					goTo( '/' );
```

Comment:
Hmm, in my testing so far, I think we could get right of the entire default block.


https://github.com/user-attachments/assets/2f49412f-cc51-4d16-aa36-26c702d458fe

Reply:
Me likes more red! Pushed an update with the whole `default` block removed